UserCard.jsx

```diff
@@ -0,0 +1,39 @@
+function UserCard({ userId }){
+  const [user, setUsers] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true)
+  
+  React.useEffect(()=>{
+    getUser(userId)  
```

Comment:
👍 - I like separating our the logic of fetching the user into a separate function. That makes this really easy to read - nice work!